garage/tf/core/cnn.py

```diff
@@ -33,56 +28,37 @@ def cnn(input_var,
                     of intermediate dense layer(s).
         hidden_b_init: Initializer function for the bias
                     of intermediate dense layer(s).
-        output_nonlinearity: Activation function for
-                    output dense layer.
-        output_w_init: Initializer function for the weight
-                    of output dense layer(s).
-        output_b_init: Initializer function for the bias
-                    of output dense layer(s).
 
     Return:
         The output tf.Tensor of the CNN.
     """
-    strides = [1, stride, stride, 1]
-
+    if padding not in ['SAME', 'VALID']:
```

Comment:
i think TensorFlow would also throw a ValueError. Any reason you want to raise the error here?

Reply:
I was not aware of that. Then I think it's fine to let TensorFlow handle it.